<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cowiek Maupa</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="editNetworkButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>Create new neural network</string>
        </property>
        <property name="text">
         <string>Edit network</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>Load picture</string>
        </property>
        <property name="text">
         <string>Change picture</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="aboutButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>See program informations</string>
        </property>
        <property name="text">
         <string>About</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>Exit program</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGraphicsView" name="originalView">
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>Original picture</string>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="restoredView">
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>Restored picture</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Epoch number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="epochDisplay">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>64</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string>Current epoch number</string>
          </property>
          <property name="numDigits">
           <number>7</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Image error:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="errorDisplay">
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string>Current image error</string>
          </property>
          <property name="numDigits">
           <number>7</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>Start restoring picture</string>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stepButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>Do next iteration and stop</string>
        </property>
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CowiekMaupa.qrc"/>
 </resources>
 <connections/>
</ui>
